### DISCUSSION TITLE
Pulling value from Budibase db

### DISCUSSION THREAD

**Discussion by amilkows** [Budibase/budibase#8379] [Help]
Newbie question. Can't figure out from docs or discussions - sorry. 
Trying to get values from a table when there is a unique solution. In this example, I am looking for MRN. The preview table shows a selected Accession Number from drop-down.  My MRN picker shows unique. I tried the approach of assigning values from actions, which didn't work.  Help on this and then a more generalized solution (if there are more than one solution, etc)?  
![image](https://user-images.githubusercontent.com/116472082/197561439-3b1464f2-3552-4d45-88f7-73fc83423cc3.png)
![image](https://user-images.githubusercontent.com/116472082/197561515-a5cb745d-f216-4949-a0cd-6444bbb32471.png)
![image](https://user-images.githubusercontent.com/116472082/197561561-d4ffacfb-59b5-4d75-a242-94a5c619ba0d.png)

**Comment by melohagan** [comment]
Hey @amilkows 

I'm afraid I don't understand the question!

**Reply by amilkows** [reply]
Sorry and thanks for the replay @melohagan .  How about:  'how do I pull values from a db'?  
In the example above, I filter the db to just one row.  Thank you in advance for your help.

**Reply by melohagan** [reply]
I'm just noticing that in the first screenshot that you have `{{ Study_Data.Fields.MRN }}` but *Study_Data* is a Data Provider. 
You can only reference Fields on a form, such as `Radiologist_Thyroid_Report.Fields`

Then you can set the schema of the update type form to point to the table you want to pull in data for, but also you'll need a repeater to get data for a single entry.

 - https://docs.budibase.com/docs/forms#form-schema
